### ISSUE TITLE
Adding interactive mapping

### ISSUE THREAD

**Issue by ifarah** [dlab-berkeley/Census-Data-in-R#11]
Also during the workshop, we observed great excitement for interactive mapping.
Maybe adding a little bit more of mapping could be exciting for folks?
Also, there was excitement for having a shiny workshop as a continuation of this workshop or a workshop where these interactive maps could be embedded within a website. I also suggested Github pages as a potential short workshop for those that want to show their academic profiles or course projects, embedding some of these interactive maps. @Averysaurus do you have any other thoughts on what we discussed the other day for the workshop?

**Comment by Averysaurus** [comment]
I agree that interactive mapping is exciting stuff to many people, @ifarah! I included one leaflet example in a pull request  this branch [here:](https://github.com/Averysaurus/Census-Data-in-R). Also yes to shiny workshop, somehow in the future i hope.
